Sources/MapboxNavigation/RouteLineController.swift

```diff
@@ -138,7 +138,11 @@ extension NavigationMapView {
             }
         }
         
-        internal func locationUpdate(newLocation: Location) {
+        func locationUpdate(newLocation: Location) {
```

Comment:
It's really hard to reproduce such crash in real life conditions.
```swift
internal func locationUpdate(newLocation: Location) {
    if routeLineTracksTraversal {
        navigationMapView.travelAlongRouteLine(to: newLocation.coordinate)
    }
}
```

can be changed to:

```swift
internal func locationUpdate(newLocation: Location) {
    DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
        if self.routeLineTracksTraversal {
            self.navigationMapView.travelAlongRouteLine(to: newLocation.coordinate)
        }
    }
}
```

to reliably reproduce it.